arcgis-ios-sdk-samples/Layers/Show labels on layers/ShowLabelsOnLayersViewController.swift

```diff
@@ -16,90 +16,68 @@ import UIKit
 import ArcGIS
 
 class ShowLabelsOnLayersViewController: UIViewController {
-    @IBOutlet private weak var mapView: AGSMapView!
+    @IBOutlet private weak var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap()
+            // Set the map viewpoint to show the layer.
+            mapView.setViewpointCenter(AGSPoint(x: -10840000, y: 4680000, spatialReference: .webMercator()), scale: 20000000)
+        }
+    }
     
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        
-        // add the source code button item to the right of navigation bar
-        (navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["ShowLabelsOnLayersViewController"]
-        
-        // create a map with a light gray canvas basemap.
+    private func makeMap() -> AGSMap {
+        // Create a map with a light gray canvas basemap.
         let map = AGSMap(basemapStyle: .arcGISLightGrayBase)
-        // assign the map to the map view
-        mapView.map = map
-        
+        return map
+    }
+    
+    /// Adds a feature layer and its labels.
+    private func addLayerAndLabels() {
         /// A URL for a feature service layer.
```

Comment:
Since we refactor this sample, please help to also clean up the comments. Use `//` instead of `///` for local variables.